pkg/utils/o11y/controller.go

```diff
@@ -31,9 +39,9 @@ func NewSuiteController(kube *kubeCl.CustomClient) (*SuiteController, error) {
 	}, nil
 }
 
-func (h *SuiteController) convertBytesToMB(bytesValue float64) float64 {
-	megabytesValue := bytesValue / (1000 * 1000)
-	return math.Round(megabytesValue*10) / 10
+func (h *SuiteController) convertBytesToMB(valueInBytes float64) int {
+	valueInMegabytes := valueInBytes / (1000 * 1000)
```

Comment:
Should it be 1000 or 1024?

Reply:
Openshift uses the SI (International System of Units) standard [1000].